app/views/publishers/vacancies/edit_vacancy_sections/_edit_important_dates.html.slim

```diff
@@ -1,5 +1,6 @@
 = render "publishers/vacancies/error_tag", attributes: %i[publish_on expires_at starts_on], section: "important_dates"
 
+#expiry_time
```

Comment:
what am i missing here, why do you need an extra div with this id when the next element has id anyway?

Reply:
i guess is for an anchor tag to link to? actually i get it